cartography/intel/gcp/loadbalancer.py

```diff
@@ -518,6 +519,7 @@ def transform_regional_url_maps(maps: List, region: str, project_id: str):
     for url_map in maps:
         url_map['region'] = region
         url_map['type'] = 'regional'
+        url_map['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='load_balancer')
```

Comment:
same as above.

Reply:
Added in consolelink project. Raised a PR